README.md

```diff
@@ -76,6 +70,24 @@ export LD_LIBRARY_PATH=/path/to/cudnn/lib:$LD_LIBRARY_PATH
 Do not forget to restart your terminal session (or `source` it) to enable these changes.
 And then, reinstall Chainer.
 
+If you want to use HDF5 serialization, please install h5py manually.
+h5py requires libhdf5.
+Anaconda distribution includes this package.
+If you are using another Python distribution, use either of the following commands to install libhdf5 depending on your Linux environment:
+
+```
+apt-get install libhdf5-dev
+yum install hdf5-devel
+```
+
+And then, install h5py via PyPI.
+You may need to install Cython for h5py.
+
+```
+pip install cython
+pip install h5py
+```
+
```

Comment:
This Installation section gets long. How about splitting it into subsections, like "Minimum installation", "Installation with CUDA", and "HDF5 Support"?